src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -82,13 +82,25 @@ class IntelliJPlugin implements Plugin<Project> {
                 configureProjectAfterEvaluate(subproject, subprojectExtension)
             }
         }
-        configureIntellijDependency(project, extension)
-        configurePluginDependencies(project, extension)
+
+        Configuration idea = project.configurations.create('idea')
+        Configuration ideaPlugins = project.configurations.create('ideaPlugins')
+
+        if (JavaPlugin.hasProperty('COMPILE_ONLY_CONFIGURATION_NAME')) {
+            project.configurations.getByName(JavaPlugin.COMPILE_ONLY_CONFIGURATION_NAME).extendsFrom ideaPlugins
```

Comment:
oh, my bad actually, I wanted to make ideaPlugins extend from idea. But I agree, I'm not sure if it makes sense, maybe better to keep the two configs separate?